tests/detect-domain-01/suricata.yaml

```diff
@@ -0,0 +1,12 @@
+%YAML 1.1
```

Comment:
Do the tests not work if this is left out? A yaml should only be added to the tests if the default config is not sufficient.

Reply:
No; will remove.